Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -178,11 +183,11 @@ public extension VisualInstruction.Component {
         /**
          Initializes a mapbox shield with the given name, text color, and display ref.
          */
-        public init(baseURL: URL, name: String, textColor: String, text: String) {
+        public init(baseURL: URL, name: String, textColor: String, code: String) {
```

Comment:
Why do we need this change? As it's a backward-incompatible one.

Reply:
It was to not interfere with `text` in the coding keys. I can actually change it back.